sdk/keyvault/Azure.Security.KeyVault.Secrets/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 4.0.2 (Unreleased)
+
+### Minor changes
+
+- `SecretClient.PurgeDeletedSecret` properly traces errors ([#9658](https://github.com/Azure/azure-sdk-for-net/issues/9658))
```

Comment:
Could you explain to me when should we include a link to an issue vs just saying what the change was?

Reply:
I'm trying to figure that out. I included this as an example and asked a few people, though with no response. Will ping them. It's common in many other first- and third-party projects, though. PowerShell Core and VSCode, for example, do a great job of this. When the bugs are opened by customers, they really appreciate the recognition for finding and even fixing them. It fosters great community engagement.

Reply:
When you find an asnwer, could you share it with the team?
This is the first time I see it, and will be good that team wise we show consistency in our changelogs.

Reply:
@adrianhall what do you think about linking issues post-GA? See my response above for context.

Reply:
Talked with Adrian offline and he agrees this is acceptable. I will open an issue to update the guidance.